bigcommerce/checkout-sdk-js
# feat(payment): PAYPAL-2908 updated braintree sdk version

## What?
Updated braintree sdk version in all places where Braintree SDK is in use

## Why?
To avoid issues with Braintree SDK version inconsistency between different Braintree integrations

## Testing / Proof
Unit tests
Manual tests

We can see that all versions are the same on the screenshot:
<img width="1512" alt="Screenshot 2023-09-04 at 10 18 51" src="https://github.com/bigcommerce/checkout-sdk-js/assets/25133454/3916dbc3-8a57-4da5-be45-33297296bb11">
